src/main/scala/vsys/blockchain/contract/ContractTokenV2.scala

```diff
@@ -0,0 +1,203 @@
+package vsys.blockchain.contract
+
+import com.google.common.primitives.Ints
+import vsys.blockchain.contract.ContractGen._
+import vsys.blockchain.state._
+import vsys.utils.serialization.Deser
+
+object ContractTokenV2 {
+  lazy val contractTokenWhiteList: Contract = Contract.buildContract(Deser.serilizeString("vdds"), Ints.toByteArray(2),
+    Seq(initFunc),
+    Seq(supersedeFunc, issueFunc, destroyFunc, updateListFunc,
+      sendWhitelistFunc, transferWhitelistFunc, depositWhitelistFunc, withdrawWhitelistFunc,
+      totalSupplyFunc, maxSupplyFunc, balanceOfFunc, getIssuerFunc),
+    Seq(issuerStateVar.arr, makerStateVar.arr),
+    Seq(listMap.arr),
+    Seq(triggerTextual, descriptorWhitelistTextual, stateVarTextual, whitelistStateMapTextual)
+  ).explicitGet()
+
+  lazy val contractTokenBlackList: Contract = Contract.buildContract(Deser.serilizeString("vdds"), Ints.toByteArray(2),
+    Seq(initFunc),
+    Seq(supersedeFunc, issueFunc, destroyFunc, updateListFunc,
+      sendBlacklistFunc, transferBlacklistFunc, depositBlacklistFunc, withdrawBlacklistFunc,
+      totalSupplyFunc, maxSupplyFunc, balanceOfFunc, getIssuerFunc),
+    Seq(issuerStateVar.arr, makerStateVar.arr),
+    Seq(listMap.arr),
+    Seq(triggerTextual, descriptorBlacklistTextual, stateVarTextual, blacklistStateMapTextual)
+  ).explicitGet()
+
+  // StateVar
+  val issuerStateVar: StateVar  = StateVar(0.toByte, DataType.Address.id.toByte)
+  val makerStateVar: StateVar   = StateVar(1.toByte, DataType.Address.id.toByte)
+  lazy val stateVarTextual: Array[Byte] = ContractPermitted.stateVarTextual
+
+  // State Map
+  val stateMapWhitelist    = List("whitelist", "userAccount", "isInList")
+  val stateMapBlacklist    = List("blacklist", "userAccount", "isInList")
+  val listMap: StateMap    = StateMap(0.toByte, DataType.Account.id.toByte, DataType.Boolean.id.toByte)
+  lazy val whitelistStateMapTextual: Array[Byte] = textualStateMap(Seq(stateMapWhitelist))
+  lazy val blacklistStateMapTextual: Array[Byte] = textualStateMap(Seq(stateMapBlacklist))
+
+  // initTrigger
+  lazy val initFunc: Array[Byte] = ContractPermitted.initFunc
+  lazy val initFuncBytes: Array[Byte] = ContractPermitted.initFuncBytes
+
+  // Functions
+  // Supersede
+  lazy val supersedeFunc: Array[Byte] = ContractPermitted.supersedeFunc
+  val supersedeFuncBytes: Array[Byte] = ContractPermitted.supersedeFuncBytes
+
+  // Issue
+  lazy val issueFunc: Array[Byte] = ContractPermitted.issueFunc
+  val issueFuncBytes: Array[Byte] = ContractPermitted.issueFuncBytes
+
+  // Destroy
+  lazy val destroyFunc: Array[Byte] = ContractPermitted.depositFunc
+  val destroyFuncBytes: Array[Byte] = ContractPermitted.depositFuncBytes
```

Comment:
typo in line 55: `depositFunc` should be `destroyFunc`
line 56 same with line 55